### DISCUSSION TITLE
How to set can zoom option?

### DISCUSSION THREAD

**Discussion by TheZoker** [fancyapps/ui#242] [Q&A] [answered]
Hi there,

How can I set, when `can-zoom-in` is allowed? On the demo page I saw, that when I shrink the page height, I can zoom in.  But when I look at the image in normal height window, the zoom is disabled. Is there an option for that?

Thanks!

**Comment by fancyapps** [comment, marked as answer]
`can-zoom-in` class name is removed when image fits inside the container. Sorry, I do not understand what are you looking for. If you want to customize it, use `Image: { canZoomInClass : '' }`.

**Reply by TheZoker** [reply]
I want to always be able to zoom an image and not to the height of the container, but further into the image

**Reply by fancyapps** [reply]
You can, for example, change max scale, like in this demo - https://fancyapps.com/playground/1C9 If you have any suggestions, I'd love to hear.
